jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -704,7 +704,219 @@ protected void deployScriptsToUserJwalaScriptsDir(Jvm jvm, User user) throws Com
+                LOGGER.info(errorMessage);
+                throw new InternalErrorException(FaultType.REMOTE_COMMAND_FAILURE, errorMessage);
+            }
+//Does this required??
+            validateJvmAndAppResources(jvm);
+
+            checkForJvmBinaries(jvm);
+
+            distributeBinaries(jvm);
+
+            // create the scripts directory if it doesn't exist
+           // createScriptsDirectory(jvm);
+
+            //REQUIRED
+            // copy the install and deploy scripts
+        //    deployScriptsToUserJwalaScriptsDir_JDKUpgrade(jvm, user);
+
+            // delete the service
+          //  deleteJvmService(jvm, user);
+
+            // create the jar file
+            //
+            //final String jvmConfigJar = generateJvmConfigJar(jvm);
+
+            // copy the jar file to the staging area
+            //secureCopyJvmConfigJar(jvm, jvmConfigJar, user);
+
+            // call script to backup and tar the current directory and
+            // then untar the new tar, needs jar
+            //deployJvmConfigJar(jvm, user, jvmConfigJar);
+
+            // copy the individual jvm templates to the destination
+            //deployJvmResourceFiles(jvm, user);
+
+            // deploy any application context xml's in the group
+            //deployApplicationContextXMLs(jvm, user);
+
+            // re-install the service
+            //installJvmWindowsService(jvm, user);
+
+            // set the state to stopped
+            updateState(jvm.getId(), JvmState.JVM_STOPPED);
+
+            didSucceed = true;
+        } catch (CommandFailureException e) {
+            LOGGER.error("Failed to upgrade JDK :: the JVM config for {}", jvm.getJvmName(), e);
+            throw new InternalErrorException(FaultType.REMOTE_COMMAND_FAILURE, "Failed to generate the JVM config: " + jvm.getJvmName(), e);
+        } finally {
+            binaryDistributionLockManager.writeUnlock(jvmName + "-" + jvm.getId().toString());
+            LOGGER.debug("End upgrade JDK for {} by user {}", jvmName, user.getId());
+
+            final EventType eventType = didSucceed ? EventType.SYSTEM_INFO : EventType.SYSTEM_ERROR;
+
+            String historyMessage = didSucceed ? "Remote JDK upgrade of jvm " + jvm.getJvmName() + " to host " + jvm.getHostName() + " succeeded" :
+                    "Remote JDK upgrade of jvm " + jvm.getJvmName() + " to host " + jvm.getHostName() + " failed";
+
+            historyFacadeService.write(jvm.getHostName(), jvm.getGroups(), historyMessage, eventType, user.getId());
+        }
+        return jvm;
+    }
+
+    /**
+     * This method will deploy necessary scripts to Jwala Directory during JDK upgrade process
+     * @param jvm
+     * @param user
+     * @throws CommandFailureException
+     * @throws IOException
+     */
+    protected void deployScriptsToUserJwalaScriptsDir_JDKUpgrade(Jvm jvm, User user) throws CommandFailureException, IOException {

```

Comment:
This method .. I just created for next step.. For now we are not using this method.We only use upgradeJDKAnd DeployJVM method

Reply:
I know all of this code is going to be changed eventually, but try to follow the Java conventions even when you're writing experimental code. It will make clean up easier in the end.